angel_system/activity_hmm/simulate_data.py

```diff
@@ -0,0 +1,486 @@
+import numpy as np
+import pandas as pd
+import glob
+import os
+from sklearn.metrics import precision_recall_curve
+from sklearn.metrics import  accuracy_score
+import scipy
+
+from angel_system.activity_hmm import ActivityHMM, get_skip_score, \
+    score_raw_detections
+from angel_system.impls.detect_activities.swinb.swinb_detect_activities import SwinBTransformer
+from angel_system.eval.support_functions import time_from_name
+from angel_system.eval.visualization import EvalVisualization
+from angel_system.eval.compute_scores import EvalMetrics
+
+try:
+    import matplotlib.pyplot as plt
+    from matplotlib.ticker import MultipleLocator, FormatStrFormatter, \
+        AutoMinorLocator
+    HAS_MATLOTLIB = True
+except ModuleNotFoundError:
+    HAS_MATLOTLIB = False
+
+
+
+class_str = ['background',
```

Comment:
Would it be good to read from the config file now?

Reply:
moved to a sandboxy scripts subdir